src/tools/quemap/writebsp.c

```diff
@@ -376,42 +378,6 @@ static void EmitBrushes(void) {
 			bs->texinfo = b->sides[j].texinfo;
 		}
 
-		// add any axis planes not contained in the brush to bevel off corners
-		for (int32_t axis = 0; axis < 3; axis++) {
-			for (int32_t side = -1; side <= 1; side += 2) {
-				// add the plane
-				vec3_t normal = Vec3_Zero();
-				normal.xyz[axis] = side;
-
-				float dist;
-				if (side == -1) {
-					dist = -b->bounds.mins.xyz[axis];
-				} else {
-					dist = b->bounds.maxs.xyz[axis];
-				}
-
-				const int32_t plane_num = FindPlane(normal, dist);
```

Comment:
So here is the late-stage beveling code. This way too late to be doing this, in my opinion. The BSP is finalized at this point, and we're just flushing it to disk. FindPlane can and does create new planes if necessary, and in this instance, those planes would never make it to the BSP. So I thought it best to relocate this code to AddBrushBevels.